build/vsoSymlinksDotNetCore.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+#--------------------------------------------------------------------------------------------------------------
+#  Copyright (c) Microsoft Corporation. All rights reserved.
+#  Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
+#--------------------------------------------------------------------------------------------------------------
+
+set -ex
+
+splitSdksDir="/opt/dotnet"
+
+allSdksDir="/home/codespace/.dotnet"
+mkdir -p "$allSdksDir"
+
+# Copy latest muxer and license files
+cp -f "$splitSdksDir/lts/dotnet" "$allSdksDir"
+cp -f "$splitSdksDir/lts/LICENSE.txt" "$allSdksDir"
+cp -f "$splitSdksDir/lts/ThirdPartyNotices.txt" "$allSdksDir"
+
+function createLinks() {
```

Comment:
why do we want to create links, what'll be the purpose for using it

Reply:
vso/codespaces team need this links to be created as per latest guidelines from the dotnetcore team to keep multiple dotnet sdk... so we need these links to standardize it